src/en/news/blog/2023/telemetry-celebrate-1-exabyte/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Celebrating one exabyte of Ceph storage!"
+date: "2023-04-28"
+author: "Yaarit Hatuka"
+image: "images/telemetry_1_eb_milestone.jpg"
+tags:
+  - "ceph"
+  - "telemetry"
+---
+
+We are excited to announce we have reached the 1 exabyte milestone of Ceph storage, reported through our telemetry by more than 2,500 Ceph clusters in the community.
+
+## Background
+
+Ceph telemetry was introduced in 2019 as a service that allows users to phone home information about their deployments. By opting-in, the telemetry module generates anonymized reports according to the [channels](https://docs.ceph.com/en/latest/mgr/telemetry/#channels) that are enabled, and sends them daily to our backend.
+
+Our [public telemetry dashboards](https://telemetry-public.ceph.com/) display aggregated time series analysis about version adoption rates, cluster storage capacity distribution, storage capacity by version, and more.
+
+Since the service was launched, we see a steady opt-in increase from community installations. Here we can see a breakdown of the total storage capacity by version over the past four years:
+
+![total_storage_capacity_by_version](images/total_storage_capacity_by_version.jpg)
+
+## Make your voice heard
+
+In addition to mailing lists, that serve as a great communication channel when looking for any feedback from the community, telemetry is an excellent resource for developers that helps answer questions about how Ceph is used. For example, when considering when to deprecate Filestore, the telemetry data helped in understanding how commonly it is being deployed vs. BlueStore across versions. Here is what telemetry tells us about OSD count by objectstore and version:
+
+![osd_by_objectstore_and_version](images/osd_by_objectstore_and_version.jpg)
+
+By looking at real data from real clusters, we can make educated decisions about the continuation and support of features. Clusters that phone home make their voice heard when we look for answers.
+
+## Privacy
+
+The telemetry reports are anonymized, and do not contain sensitive information like host, pool, or object names. The FSID of the cluster is not reported, and IPs are not stored.
+See a sample report with:
+```
+ceph telemetry preview-all
+```
+from Quincy, or with:
+```
+ceph telemetry show-all
+```
+before Quincy.
```

Comment:
https://telemetry-1-exabyte-blog-post.ceph.io/en/news/blog/2023/telemetry-celebrate-1-exabyte/#privacy is looking a bit unintuitive. How about restructuring it to look like: 


See a sample report in Quincy and later releases with:
```
ceph telemetry preview-all
```

and in prior releases with:
```
ceph telemetry show-all
```

Reply:
@neha-ojha thanks, fixed!